### PR TITLE
Get information about a container service

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,11 +191,231 @@ Example (the following are equivalent):
...
+        "name": "api",
+        "env": {
+          "FOOFOOFOO": "kJBDGKLBEGLKWBGLsndlkFNFGLKEN",
+          "dKJGBLSGB": "LDNGLWK"
+        },
+        "portMappings": [
+          "8080/tcp"
+        ],
+        "secrets": {
+          "DERPDERP": "MySecretString"
```
Comment (tenyo) [inline README.md:321]: Are secrets going to be displayed?

File: cmd/get_container.go [hunk 2]
```diff
@@ -77,74 +103,230 @@ func container(id string) ([]byte, error) {
 		return []byte{}, err
 	}
 
-	return resourceSummary(resource, size, info.Status)
+	log.Debugf("%+v", info)
```
Comment (tenyo) [inline cmd/get_container.go:106]: Where do these log entries end up?

File: pkg/spinup/container.go [hunk 3]
```diff
@@ -6,15 +6,165 @@ import (
...
+		StoppedReason     string
+		StoppingAt        string
+		Tags              []*NameValue
+		TaskArn           string
+		TaskDefinitionArn string
+		Version           int64
+	}
+}
+
+type ContainerDefinition struct {
```
Comment (tenyo) [inline pkg/spinup/container.go:106]: Wow, big structs 😮 If any of those keys change in AWS this shouldn't break, right? Say, `Memory` changes to `MemorySize`, then the CLI is just going to return an empty value for `Memory`, I think.

File: README.md (hunk 1 above)
Comment (fishnix) [reply README.md:321]: just the name - the API doesn't return the values

File: cmd/get_container.go (hunk 2 above)
Comment (fishnix) [reply cmd/get_container.go:106]: stdout right now

File: pkg/spinup/container.go (hunk 3 above)
Comment (fishnix) [reply pkg/spinup/container.go:106]: right - should jus tbe empty

**Comment by damnski** [comment]
lgtm

**Pull request merged**